### PR TITLE
Added VERSION and CHANGELOG

### PR DESCRIPTION
**Pull request by GragonGit**
Wait for #4 to be closed before merging

### CODE REVIEW

File: VERSION
Comment (TanConan) [inline VERSION:1]: Is this file needed?
If not, please delete it, as we already track the mod version in the gradle.properties file

Comment (GragonGit) [reply VERSION:1]: Yesn't. Needed not really but it gives easy access especially for DevOps stuff and there are ways to write a gradle task that automatically synchronizes the version number. But I can delete it for now and we'll wait until Retoriax complains

Comment (TanConan) [reply VERSION:1]: Shouldn't be a problem to automatically change the version number in the gradle.properties file I guess? We will see

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Changelog
+The Arcane Artistry Changelog. We use Epoch-Based Versioning where:
+- `X` is for major releases
+- `Y` is for new features
+- `Z` is for bug fixes, repository changes and code maintenance
```
Comment (TanConan) [inline CHANGELOG.md:5]: repository changes and code maintenance?
I think its weird to update the version number to include changes that dont affect the mod.jar at all.
Users would suggest something got fixed with the mod which can be confusing I would say.
So, do we only update "Z" if a bug is fixed or for example a texture got updated (without changing file name)?

**Review by TanConan — APPROVED** [review]
